package.json

```diff
@@ -23,7 +23,16 @@
     "express": "^4.16.4",
     "path": "^0.12.7",
     "serve-favicon": "^2.5.0",
-    "mongodb": "^3.1.10"
+    "mongodb": "^3.1.10",
+    "parcel": "^1.12.3",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-testing-library": "^7.0.0",
```

Comment:
1. React and anything related to the frontend belongs in the client package.json only.
1. Anything to do with bundling and testing should be in dev dependencies, so react-testing-library, parcel and all babel packages.